root/static/scripts/edit/URLCleanup.js

```diff
@@ -1041,12 +1184,19 @@ const CLEANUPS = {
         const prefix = m[1];
         switch (id) {
           case LINK_TYPES.otherdatabases.artist:
-            return {result: prefix === 'people'};
+            return {
+              result: prefix === 'people',
+              target: ERROR_TARGETS.RELATIONSHIP,
+            };
           case LINK_TYPES.otherdatabases.recording:
-            return {result: prefix === 'files'};
+            return {
+              result: prefix === 'files',
+              target: ERROR_TARGETS.RELATIONSHIP,
```

Comment:
There's a ton of `target: ERROR_TARGETS.RELATIONSHIP` additions - would it maybe make sense to have this one as default, and only add a target if it should be a different one?

Reply:
Indeed. But @yvanzo said:
> The default should probably be no value though as it might affect either URL or relationship or both.

Reply:
Having a catch-all for default value is unfortunately more verbose but safer than having the most common value.